back/engines/commercial/bulk_import_ideas/spec/acceptance/import_ideas_spec.rb

```diff
@@ -265,19 +307,19 @@ def create_project_bulk_import_ideas_xlsx
   end
 
   def create_project_bulk_import_ideas_pdf
-    base_64_content = Base64.encode64 File.read('/cl2_back/engines/commercial/bulk_import_ideas/spec/fixtures/testscan.pdf')
+    base_64_content = Base64.encode64 Rails.root.join('engines/commercial/bulk_import_ideas/spec/fixtures/testscan.pdf').read
     "data:application/pdf;base64,#{base_64_content}"
   end
 
   def create_project_bulk_import_raw_text_array
-    ["Page 1\n" \
-     "Full name\nBob Test\n" \
+    ["Full name\nBob Test\n" \
```

Comment:
We can remove this (although it should be ignored by the parser anyway at this point)

Reply:
This is needed as it is being used to mock the return value from the Google Form Parser so that we don't call google and we always get a consistent response back from the service.

Reply:
Oh, you mean name & email, not the whole thing! Will do